README.md

```diff
@@ -85,3 +97,41 @@ TF_ACC=1 go test ./builtin/providers/aws -v -run=Vpc -timeout 90m
 The `TEST` variable is required, and you should specify the folder where the provider is. The `TESTARGS` variable is recommended to filter down to a specific resource to test, since testing all of them at once can take a very long time.
 
 Acceptance tests typically require other environment variables to be set for things such as access keys. The provider itself should error early and tell you what to set, so it is not documented here.
+
+### Cross Compilation
```

Comment:
I'd also call out that this is the correct section to read if you're looking to build for distribution, as I think that's a use case people might be searching for.

So perhaps `Cross Compilation / Building for Distribution`?